apps/api/src/app/support/support.controller.ts

```diff
@@ -1,127 +1,22 @@
   ) {}
 
   @Post('plain/cards')
-  async getPlainCards() {
-    return {
-      data: {},
-
-      cards: [
-        {
-          key: 'plain-customer-details',
-          components: [
-            {
-              componentSpacer: {
-                spacerSize: 'S',
-              },
-            },
-            {
-              componentRow: {
-                rowMainContent: [
-                  {
-                    componentText: {
-                      text: 'Registered at',
-                      textColor: 'MUTED',
-                    },
-                  },
-                ],
-                rowAsideContent: [
-                  {
-                    componentText: {
-                      text: '7/18/2024, 1:00 PM',
-                    },
-                  },
-                ],
-              },
-            },
-            {
-              componentSpacer: {
-                spacerSize: 'M',
-              },
-            },
-            {
-              componentRow: {
-                rowMainContent: [
-                  {
-                    componentText: {
-                      text: 'Last signed in',
-                      textColor: 'MUTED',
-                    },
-                  },
-                ],
-                rowAsideContent: [
-                  {
-                    componentText: {
-                      text: '10/20/2024, 12:57 PM',
-                    },
-                  },
-                ],
-              },
-            },
-            {
-              componentSpacer: {
-                spacerSize: 'M',
-              },
-            },
-            {
-              componentRow: {
-                rowMainContent: [
-                  {
-                    componentText: {
-                      text: 'Last device used',
-                      textColor: 'MUTED',
-                    },
-                  },
-                ],
-                rowAsideContent: [
-                  {
-                    componentText: {
-                      text: 'iPhone 13 🍎',
-                    },
-                  },
-                ],
-              },
-            },
-            {
-              componentSpacer: {
-                spacerSize: 'M',
-              },
-            },
-            {
-              componentRow: {
-                rowMainContent: [
-                  {
-                    componentText: {
-                      text: 'Marketing preferences',
-                      textColor: 'MUTED',
-                    },
-                  },
-                ],
-                rowAsideContent: [
-                  {
-                    componentText: {
-                      text: 'Opted out 🙅',
-                    },
-                  },
-                ],
-              },
-            },
-          ],
-        },
-      ],
-    };
+  async getUserOrganizations(@Body() body: PlainCardRequestDto) {
```

Comment:
How authorization happening here? We need to verify the HMAC using their signing key and enforce it.

Reply:
@jainpawan21 but where is the logic for the validation is happening? Maybe I'm missing something 🤔

Reply:
NVM, found it 😅